float32 x_pos
float32 y_pos
float32 z_pos
float32 theta_pos
float32 grabber_pos

float32 x_torque
float32 y_torque
float32 z_torque
float32 theta_torque
float32 grabber_torque
